cypress/e2e/tenure.feature

```diff
@@ -1,8 +1,11 @@
+@tenure
 @authentication
 @common
-@processes
 @root
-@tenure
+@search
+@worktray
+@processes
+@personal-details
```

Comment:
Seems like this spec is about the display of the tenure data.

Do we really need to run it any time a new process is added, any time the search result cards have styling tweaked, or when a person receives more info fields about them?